### PR TITLE
Better reporting of errors from bitrise

### PR DESCRIPTION
**Pull request by cobbal**
refs: MBL-13568
affects: none
release note: none

### CODE REVIEW

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Danger failed to run `dangerfile.js`.
  </td>
    </tr>
  </tbody>
</table>



## Error ReferenceError
```
reportFailures is not defined
ReferenceError: reportFailures is not defined
    at Object.<anonymous> (/Users/vagrant/git/scripts/coverage/dangerfile-utils.js:20:26)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.customModuleHandler (/Users/vagrant/git/node_modules/danger/distribution/runner/runners/inline.js:106:28)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (dangerfile.js:22:43)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
```
### Dangerfile
```
15| // You should have received a copy of the GNU Affero General Public License
16| // along with this program.  If not, see <https://www.gnu.org/licenses/>.
17| //
18| 
19| // Removed require
----------------------------^
20| const fs = require('fs')
21| const path = require('path')
22| const { checkCoverage, reportFailures } = require('./scripts/coverage/dangerfile-utils')
23| const { check } = require('./scripts/update-headers')
```
  
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by cobbal** [comment]
Examples of failures:

<details><summary>:x: Swift lint</summary>
<pre>⚠️  /Users/vagrant/git/Student/Student/Routes.swift:345:6: Trailing Comma Violation: Multi-line collection literals should have trailing commas. (trailing_comma)
Linting 'DashboardEmptyTests.swift' (114/137)
Linting 'PeopleTests.swift' (115/137)</pre>
</details>
<details><summary>:x: CoreTests/IDTests/testIntLiteral</summary>
<pre>Assertion Failure: IDTests.swift:41: XCTAssertEqual failed: ("2") is not equal to ("3")</pre>
</details>

**Comment by bluwave** [comment]
This is so cool, i'm so excited for this.  Thank you.

File: scripts/build_automation/summarize-test-results.swift [hunk 1]
```diff
@@ -24,7 +24,7 @@
 //   swift sh edit summarize-test-results.swift
 
 import Foundation
-import XCResultKit // @cobbal == dev
+import XCResultKit // @davidahouse
```
Comment (bluwave) [inline scripts/build_automation/summarize-test-results.swift]: who is this?

Comment (cobbal) [reply scripts/build_automation/summarize-test-results.swift]: It points swift-sh to the dependency at https://github.com/davidahouse/XCResultKit

**Pull request merged**